frank/geometry.py

```diff
@@ -217,6 +219,11 @@ def fit(self, u, v, V, weights):
             Weights on the visibilities
         """
 
+        if all(x == 0 for x in (self._inc, self._PA, self._dRA, self._dDec)):
+            logging.info("      N.B.: All geometry parameters are 0 --> No geometry"
+                         " correction will be applied to the visibilities"
+                         )
+
```

Comment:
I'm against having this here - you can't get all zeros by accident, you have to have passed them in. If you want to check in fit.py, fine, but not here.

Reply:
Moved to fit.py